pkgs/top-level/python-packages.nix

```diff
@@ -5319,6 +5319,30 @@ let
   };
 
 
+  hidapi =
+  let rev = "78722d3c6c028cd66e3ecc1ee9fa03e63e671de9"; in
+  buildPythonPackage rec {
+    name = "hidapi-${builtins.substring 0 7 rev}";
+    buildInputs = [ self.cython pkgs.libusb1 ];
+    # Use fetchgit to retrieve submodules
+    src = pkgs.fetchgit {
+      url = https://github.com/trezor/cython-hidapi.git;
+      inherit rev;
+      sha256 = "0h093m351yzyl22zg5f4smi21d030diizc1dyq5c6ksjzchxdirl";
+    };
+    postPatch =
+      ''
+        substituteInPlace setup.py \
+            --replace /usr/include/libusb-1.0 ${pkgs.libusb1}/include/libusb-1.0
+       '';
+    meta = with stdenv.lib;
+      { description = "A Cython interface to the HIDAPI from https://github.com/signal11/hidapi";
+        license = licenses.free;
```

Comment:
Not that it matters too much, I suppose, but hidapi is available under one of 3-clause BSD, GPL3, or their custom license.